src/app/article-data.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { ArticleFormat } from './format';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ArticleDataService {
+  articleData : ArticleFormat = null;
+  constructor() { }
+
+  setObject(refrenceObject){
+    this.articleData = refrenceObject;
+  }
+
+  getObject(){
```

Comment:
Couple of changes to be made

- `setObject` and `getObject` function names do not actually convey the purpose of what they do. Rename the functions to `setArticleData()` and `getArticleData(): Article`
- Rename the parameter `referenceObject` to `article` or `articleData` or `articleList`
- Also mention the type of the parameter being passed to the above functions

Reply:
**Changes Implemented**